Swiftification/ObserverSet.swift

```diff
@@ -124,7 +124,7 @@ extension ObserverSet: CustomStringConvertible {
             entry in
             (entry.object === self
                 ? "\(entry.f)"
-                : "\(entry.object) \(entry.f)")
+                : "\(String(describing: entry.object)) \(entry.f)")
```

Comment:
This will log "Optional(...)". Because `entry.object === self` at this point, we know that `entry.object` is not optional. Try this instead:

```swift
if let object = entry.object, object === self {
    return "\(object) \(entry.f)"
} else {
    return "\(entry.f)"
}
```